contracts/implementations/tableland/DeviceDefinitionTable.sol

```diff
@@ -72,14 +74,21 @@ contract DeviceDefinitionTable is AccessControlInternal {
             revert TableAlreadyExists(manufacturerId);
         }
 
-        string memory statement = string(
-            abi.encodePacked(
-                "CREATE TABLE _",
-                Strings.toString(block.chainid),
-                "(id TEXT PRIMARY KEY, model TEXT NOT NULL, year INTEGER NOT NULL, metadata TEXT, UNIQUE(model,year))"
-            )
+        // string memory statement = string(
+        //     abi.encodePacked(
+        //         "CREATE TABLE _",
+        //         Strings.toString(block.chainid),
+        //         "(id TEXT PRIMARY KEY, model TEXT NOT NULL, year INTEGER NOT NULL, metadata TEXT, UNIQUE(model,year))"
+        //     )
+        // );
+        //uint256 tableId = tablelandTables.create(address(this), statement);
+        
+        uint256 tableId = TablelandDeployments.get().create(
+            address(this),
+            SQLHelpers.toCreateFromSchema(
+                "id TEXT PRIMARY KEY, model TEXT NOT NULL, year INTEGER NOT NULL, metadata TEXT",
```

Comment:
- is it possible to do like `UNIQUE(model,year)` 
- what about the "legacyId" ie. the ksuid for DD's we use currently